task/host-sp-comms/src/bsp/gimlet_bcde.rs

```diff
@@ -104,16 +99,15 @@ impl ServerImpl {
             //
             // Sharkfin connectors start at J206 and are numbered sequentially
             20..=29 => {
-                let (designator, f): ([u8; 7], _) =
-                    Self::get_sharkfin_vpd(index as usize - 20);
+                let dev = Self::get_sharkfin_vpd(index as usize - 20);
+                let dev_id = dev.component_id().as_bytes();
                 let mut name = *b"_______/ID";
-                name[0..7].copy_from_slice(&designator);
-                self.read_eeprom_barcode(sequence, &name, f)
+                name[0..7].copy_from_slice(&dev_id[0..7]);
+                self.read_eeprom_barcode(sequence, &name, dev)
             }
             30..=39 => {
-                let (designator, f): ([u8; 7], _) =
-                    Self::get_sharkfin_vpd(index as usize - 30);
-                self.read_at24csw080_id(sequence, &designator, f)
+                let dev = Self::get_sharkfin_vpd(index as usize - 14);
```

Comment:
so...this somehow changed from `- 30` to `- 14` and caused a panic, which...how the heck did that happen? anyway, will fix that